src/ApiFunction.ts

```diff
@@ -7,7 +6,10 @@ import { Construct } from 'constructs';
 import { LambdaReadOnlyPolicy } from './iam/lambda-readonly-policy';
 import { Statics } from './statics';
 
+type T = Lambda.Function;
+
 export interface ApiFunctionProps {
+  apiFunction: {new(scope: Construct, id:string, props?: Lambda.FunctionProps): T };
```

Comment:
Toch geen betere oplossing kunnen vinden? haha

Reply:
Het zit nu in de properties ipv in de constructor dat is misschien wel iets netter inderdaad.

Reply:
Nee, wel wat mee gespeeld. Dit scheelt weer iets ;).